LEMS/LEMS_DataEntry_L1.py

```diff
@@ -671,6 +722,154 @@ def on_bias_okay(self):
                 self.gas_cal.update_gas_check('zero_Gas_Drift_Check_CO2', 'INVALID', 'red')
             success = 0
 
+            #########################################################################
+            #gravimetric flowrate checks
+            #hp
+            #A
+            try:
+                initial = float(self.data['GravFlow_A_Initial_hp'])
+                final = float(self.data['GravFlow_A_Final_hp'])
+
+                percent_diff = (abs(initial-final) / ((initial+final) / 2)) * 100
```

Comment:
can this math and logic be in a different script and then the gui reads the pass/fail output?